cypress/e2e/task/shareAndImportTaskFromCourseDetailInSameSchool.feature

```diff
@@ -0,0 +1,77 @@
+@regression_test
+@stable_test
+@group-V
+
+Feature: Task - Teacher shares a task with another teacher from the same school
+
+    As a teacher
+    I want to share a task from my course with another teacher in the same school
+    So that the other teacher can import the task into their own course as a draft
+
+    Scenario Outline: Teacher shares a task and another teacher imports it into their course
+
+        # Step 1: # pre-condition: users creation and login in same school
+        Given I am logged in as a '<teacher_2>' at '<namespace>'
+        Given I am logged in as a '<teacher_1>' at '<namespace>'
+
+        # Step 2: Teacher 1 creates course_source and task
+        Given a course named '<course_source>' exists
```

Comment:
maybe better to use some other parameter name, `course_source` will mean something else

Reply:
done!